src/components/Icon/index.tsx

```diff
@@ -0,0 +1,8 @@
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const Icon = (color: any) => {
+  return (
+    <div className={`h-6 w-6 ${color}`} data-testid="icon">
+      Ico
+    </div>
+  );
+};
```

Comment:
We could use some TODO comment to replace this component with a proper one. Can also add a link to the task, like:
```ts
// TODO: Replace the component with a proper one in https://github.com/CodersCrew/coderscrew-website/issues/33
```